services/testing-framework/claudie_test.go

```diff
@@ -32,9 +35,49 @@ const (
 	maxTimeoutSave = 60 * 12 // max allowed time for config to be found in the database
 )
 
+var (
+	//get env var from runtime directly so we do not pollute original envs package by unnecessary variables
```

Comment:
`//abc` -> `// abc`

Reply:
There's little to no value in the comment formatting change. Isn't there?

Reply:
I don't agree, the formatting is inconsistent.
also, have a look at literally all comments in e.g. https://tip.golang.org/doc/comment, they're all `// abc` style.
I agree that it's inconsequential to how the program runs, however, people are going to read the code, which means there's value in formatting.